package.json

```diff
@@ -70,6 +71,7 @@
     "hooks": {
       "init": [
         "./src/hooks/init/twilio-api",
+        "./src/hooks/init/buy-phone-number",
```

Comment:
Is it time to think about a sub-folder structure? Just thinking it may get confusing as we add many more of these helpers.

Reply:
It's broken out by hook name now. I think that's fine for now.